app/controllers/application_controller.rb

```diff
@@ -13,12 +13,6 @@ def self.default_url_options
 
   def canonical_url(locale=I18n.locale)
     # Leave the locale out of canonical URLs in the default locale (works ok either way but needs to be consistent)
-    url_for(only_path: false, locale: locale == I18n.default_locale ? nil : locale)
-  end
-
-  def hreflang_url(locale=nil)
-    # The default hreflang URL shouldn't include locale but all other hreflang links should
-    # So unlike canonical_url we will not replace the default locale with nil
```

Comment:
going back to a simpler approach where both canonical and hreflang URLs will always have the locale included so no need to have separate methods